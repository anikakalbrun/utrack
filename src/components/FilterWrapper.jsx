import React from 'react';
import {string, node} from 'prop-types';

FilterWrapper.propTypes = {
    label: string,
    children: node
};

export default function FilterWrapper({label, children}) {
    return (
        <div className='col-sx-12 col-md-4 col-lg-3'>
            <div className='filter p-50'>
                <strong>{label}:</strong>
                {children}
            </div>
        </div>
    );
}