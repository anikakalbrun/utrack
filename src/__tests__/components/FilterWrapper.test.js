import React from 'react';
import FilterWrapper from '../../components/FilterWrapper';

let wrapper;
const props = {
    label: "Minimal spend",
    children: <span>test</span>
};

describe('FilterWrapper', () => {

    it('renders correctly', () => {
        wrapper = renderer.create(<FilterWrapper {...props}/>).toJSON();
        expect(wrapper).toMatchSnapshot();
    });

});
