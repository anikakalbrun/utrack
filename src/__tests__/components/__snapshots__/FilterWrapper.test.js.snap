// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterWrapper renders correctly 1`] = `
<div
  className="col-sx-12 col-md-4 col-lg-3"
>
  <div
    className="filter p-50"
  >
    <strong>
      Minimal spend
      :
    </strong>
    <span>
      test
    </span>
  </div>
</div>
`;
